### PR TITLE
chore(readme): tidy up tech test expectations and provides clarity

### PR DESCRIPTION
**Pull request by garryyao**
Clarify the expectations we set on SE test - option 2
 - making it clear that we'd expect the same from "what are we looking for section" for both option
 - tidy up some outdated stuff 
 - removal UK specific terms

### CODE REVIEW

**Review by grahamlicence — APPROVED** [review]

File: se-test/README.md [hunk 1]
```diff
@@ -1,18 +1,28 @@
 # Overview
 This task is intended for people who interview for our Software Engineer roles.
 
-Please make sure you read [our expectations](../README.md#what-we-are-looking-for) first before starting the following tasks:  
+Please make sure you read [our expectations](../README.md#what-we-are-looking-for) first before starting on any of the below options:
```
Comment (katemont) [inline se-test/README.md:4]: 'either of'

File: README.md [hunk 2]
```diff
@@ -19,9 +19,8 @@ Send the work to your **point of contact**.
 These points serve as a guide on what we look for in a code submission. Not all points have to be met
  - The solution fulfils the brief
  - There is some attempt at creating documentation
- - There is a README that instructs how to run the app
- - The app runs when you follow the README, or `npm start`, or run the iOS app
- - There are reasonably thorough tests
+ - There is a README that instructs how to run the app locally
+ - There are reasonably thorough tests coverage of your code (this can be unit/integration/e2e test cases)
```
Comment (katemont) [inline README.md:23]: 'There is reasonably thorough test coverage...'

File: se-test/README.md [hunk 3]
```diff
@@ -1,18 +1,28 @@
...
 
+## Option 2
+
+If you have some existing code that you would be prepared to share with us, we can look at that instead. The code should
+meet the following criteria:
+
+* It should be similar hours of your own work as with Option 1 (as opposed to others' or your team's work for example)
+* Ideally, it should be a web/mobile app that involves both client and server-side skills
+* It should demonstrate how you approach a problem with a simple solution that you're confident to walk us through in an interview scene (as opposed to sharing an app of 10k LOC for example) 
+* It should be something that's legit to share with us (as opposed to NDA protected stuff for example)
```
Comment (katemont) [inline se-test/README.md:24]: How about: 'It should be something you are permitted to share with us (not confidential or protected by NDA for example)' ?

**Comment by garryyao** [comment]
Hi @katemont thanks for correcting those looks legit for me - would you mind to please open a new PR

**Comment by katemont** [comment]
@garryyao PR [here](https://github.com/conde-nast-international/cnid-tech-tests/pull/22)

**Pull request merged**